libraries/botframework-connector/src/auth/governmentCloudBotFrameworkAuthentication.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { AuthenticationConfiguration } from './authenticationConfiguration';
+import { CallerIdConstants } from 'botframework-schema';
+import { GovernmentChannelValidation } from './governmentChannelValidation';
+import { GovernmentConstants } from './governmentConstants';
+import { ParameterizedBotFrameworkAuthentication } from './parameterizedBotFrameworkAuthentication';
+import { ServiceClientCredentialsFactory } from './serviceClientCredentialsFactory';
+import { VerifyOptions } from 'jsonwebtoken';
+import { governmentEmulatorAuthValidator } from './emulatorValidation';
+import { governmentSkillAuthValidator } from './skillValidation';
+import { makeAuthValidator } from './authValidator';
+
+export class GovernmentCloudBotFrameworkAuthentication extends ParameterizedBotFrameworkAuthentication {
+    constructor(
+        credentialFactory: ServiceClientCredentialsFactory,
+        authConfiguration: AuthenticationConfiguration,
+        verifyOptions?: Partial<VerifyOptions>,
+        openIdMetadataUrl?: string
```

Comment:
Do we need to support a custom open ID metadata URL for every auth scenario (i.e., general, emulator, skill)?